docs/release-notes/3.0.0_Beta.md

```diff
@@ -10,6 +10,14 @@ It enables you to observe the health of your database systems, explore new patte
 
 ## Release highlights
 
+### Encryption of sensitive data
+
+To strengthen the security of your monitoring setup, all sensitive information stored in the PMM Server and Client databases, including usernames, passwords, AWS keys, Azure credentials, and TLS/SSL certificates, is now encrypted.
```

Comment:
I would remove "the Client database" because all data is stored in PMM server databases